### PR TITLE
Feature/graphql updates integration

### LINKED ISSUE #169: Order Updates on Mobile App

### PR DESCRIPTION
**Pull request by driels97**
This PR integrates most of the App main features to the data available on the backend through GraphQL subscriptions:

- Orderbook;
- Balance;
- Order and Trade History;
- Graph;
- Tickers.

Closes #169, #172, #188

### CODE REVIEW

File: lib/common/orderbook/presentation/cubit/orderbook_cubit.dart [hunk 1]
```diff
@@ -26,24 +29,23 @@ class OrderbookCubit extends Cubit<OrderbookState> {
...
+        if (currentState is OrderbookLoaded) {
+          final currentOrderbook = currentState.orderbook as OrderbookModel;
+          emit(
+            OrderbookLoaded(
+              orderbook: currentOrderbook.update(putsList, delsList),
+            ),
+          );
+        }
+      },
+      onMsgError: (_) {},
```
Comment (serhii-temchenko) [inline lib/common/orderbook/presentation/cubit/orderbook_cubit.dart:47]: Isn't error should be logged or bubbled to notify that request/message receiving failed? Or this is interface requirement and this case unreachable?

File: lib/common/trades/presentation/cubits/order_history_cubit/order_history_cubit.dart [hunk 2]
```diff
@@ -33,6 +37,36 @@ class OrderHistoryCubit extends Cubit<OrderHistoryState> {
...
+          } else {
+            emit(
+              OrderHistoryLoaded(
+                  orders: [newOrder, ...currentState.orders],
+                  orderIdsLoading: currentState.orderIdsLoading),
+            );
+          }
+        }
+      },
+      onMsgError: (_) {},
```
Comment (serhii-temchenko) [inline lib/common/trades/presentation/cubits/order_history_cubit/order_history_cubit.dart:67]: Same as above.

File: lib/features/coin/presentation/cubits/trade_history_cubit/trade_history_cubit.dart [hunk 3]
```diff
@@ -29,6 +33,20 @@ class TradeHistoryCubit extends Cubit<TradeHistoryState> {
...
+      onMsgReceived: (newTrade) {
+        final currentState = state;
+
+        if (currentState is TradeHistoryLoaded) {
+          final newList = [newTrade, ...currentState.trades];
+
+          emit(TradeHistoryLoaded(trades: newList));
+        }
+      },
+      onMsgError: (_) {},
```
Comment (serhii-temchenko) [inline lib/features/coin/presentation/cubits/trade_history_cubit/trade_history_cubit.dart:47]: Same as above.

File: lib/features/landing/presentation/cubits/ticker_cubit/ticker_cubit.dart [hunk 4]
```diff
@@ -22,7 +26,29 @@ class TickerCubit extends Cubit<TickerState> {
...
+              final currentState = state;
+
+              if (currentState is TickerLoaded) {
+                final newMap = {...currentState.ticker}..[newTicker.m] =
+                    newTicker;
+
+                emit(TickerLoaded(ticker: newMap));
+              }
+            },
+            onMsgError: (_) {},
```
Comment (serhii-temchenko) [inline lib/features/landing/presentation/cubits/ticker_cubit/ticker_cubit.dart:48]: Same as above.

File: lib/features/trade/presentation/cubits/coin_graph_cubit.dart [hunk 5]
```diff
@@ -25,6 +29,28 @@ class CoinGraphCubit extends Cubit<CoinGraphState> {
...
+          emit(
+            CoinGraphLoaded(
+              timestampSelected: newTimestampChart,
+              dataList: newGraphList,
+              indexPointSelected: null,
+            ),
+          );
+        }
+      },
+      onMsgError: (_) {},
```
Comment (serhii-temchenko) [inline lib/features/trade/presentation/cubits/coin_graph_cubit.dart:51]: Same as above.

**Comment by driels97** [comment]
@serhii-temchenko should be solved, please check again

**Comment by serhii-temchenko** [comment]
Some tests are broken for Android build after refactoring. Please check CI.

**Comment by driels97** [comment]
@serhii-temchenko fixed

**Review by serhii-temchenko — APPROVED** [review]

**Pull request merged**